md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/DQSetupAPI.java

```diff
@@ -262,8 +234,7 @@ RestWrapper insert(@ModelAttribute("dqsetup")
      * @return restWrapper Updated instance of DQSetup instance.
      */
     @RequestMapping(value = {"/", ""}, method = RequestMethod.POST)
-    public
-    @ResponseBody
+    @ResponseBody public
     RestWrapper update(@ModelAttribute("dqsetup")
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "update" is 15 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)